.github/workflows/solidity.yml

```diff
@@ -66,7 +66,7 @@ jobs:
         run: yarn
 
       - name: Compile contracts and generate typechain bindings
-        run: yarn compile
+        run: cp -f .env.dist .env && yarn compile
```

Comment:
compile won't run if .env file is not present or is missing variables. Since solc wants to hydrate all the used env variables at compile time.